server/models/db/workerTimeslot.js

```diff
@@ -8,10 +8,7 @@ const TimeslotStatus  = require('../../constants/timeslotStatus.json');
 // This method inserts an appointment entry given specific information.
 async function insertWorkerTimeslot(slotId, schoolId, workerId, status, date) {
     // Added conditional statements to prevent status and date from storing "null" instead of null. 
-    // Date is currently being input as a string. Unsure of correct format for date data type. 
```

Comment:
Hi Peter,
Is there a specific reason as to why these comments were removed?

Reply:
Noticed that it was unnecessary and an artefact from a complete development process. I am aware that it is not relevant to the current ticket. I am comfortable with discarding this change if it follows better practices